base/ca/src/com/netscape/ca/CAService.java

```diff
@@ -922,62 +924,80 @@ private X509CertImpl issueX509Cert(
 
                         // loop through all CT log servers
                         for (LogServer ls : logServers) {
-                            logger.debug("Processing log server ID: " + ls.getId());
+                            logger.debug(method + "Processing log server ID: " + ls.getId());
 
                             String ct_host = ls.getUrl().getHost();
-                            logger.debug("Log server host: " + ct_host);
+                            logger.debug(method + "Log server host: " + ct_host);
 
                             int ct_port = ls.getUrl().getPort();
-                            logger.debug("Log server port: " + ct_port);
+                            logger.debug(method + "Log server port: " + ct_port);
 
                             // TODO: Refactor to form right rest API
                             String ct_uri = ls.getUrl() + "ct/v1/add-pre-chain";
-                            logger.debug("Log server URI: " + ct_uri);
+                            logger.debug(method + "Log server URI: " + ct_uri);
 
                             String respS = certTransSendReq(
                                     ct_host, ct_port, ct_uri, ctRequest);
-                            logger.debug("Response from log server " + respS);
-
-                            // verify the sct:
-                            verifySCT(CTResponse.fromJSON(respS), tbsCert, ls.getPublicKey());
-
-                            ctResponses.add(respS);
+                            if (respS == null) {
+                                errMsg = method + "Response from CT log server null";
+                                logger.debug(errMsg);
+                                // skip to next CT log server
+                                //throw new EBaseException(errMsg);
+                                continue;
+                            }
+                            logger.debug(method + "Response from CT log server " + respS);
+
+                            // verify the sct
+                            Boolean verified =
+                                    verifySCT(CTResponse.fromJSON(respS), tbsCert, ls.getPublicKey());
+                            if (verified)
+                                ctResponses.add(respS);
+                            else { // not verified
+                                // skip to next CT log server
```

Comment:
I believe in the future we won't be issuing a certificate if CT verification fails. In that case, instead of skipping do you want to report an error? We can throw an exception in the future. For now, we can report an error